cfme-performance/utils/providers.py

```diff
@@ -286,6 +287,20 @@ def add_provider(provider):
 
     logger.debug('Added Provider: {}, Response: {}'.format(provider['name'], response))
 
+    # Workaround for Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1351253
+    if provider['type'] == "ManageIQ::Providers::Microsoft::InfraManager":
+        print "In if statement!!!"
+        time.sleep(10)
+        logger.debug('Running rails command for Microsoft InfraManager Workaround')
+        ssh_client = SSHClient()
+        command = ('m = ExtManagementSystem.find_by_name "' + provider['name'] +
```

Comment:
My preference is on using the .format to put things into strings.  An example you might find useful for this case would be here:

https://github.com/akrzos/cfme-performance/blob/master/cfme-performance/utils/appliance.py#L279-L281